manic/checkout.py

```diff
@@ -216,9 +216,11 @@ def commandline_arguments(args=None):
                         '%(prog)s. By default only summary information '
                         'is provided. Use verbose output to see details.')
 
-    parser.add_argument('-v', '--verbose', action='store_true', default=False,
+    parser.add_argument('-v', '--verbose', action='count', default=0,
                         help='Output additional information to '
-                        'the screen and log file.')
+                        'the screen and log file. For --status, this flag '
+                        'can be used a second time, increasing the verbosity'
```

Comment:
need a space after verbosity

Reply:
This is addressed in https://github.com/bandre-ucar/manage_externals/pull/2 if it is accepted.